testutil/proof/fixture_generators.go

```diff
@@ -1,19 +1,27 @@
 package proof
 
 import (
+	"crypto/sha256"
 	"encoding/binary"
 	"math/rand"
 	"testing"
 
-	"github.com/pokt-network/smt"
 	"github.com/stretchr/testify/require"
 
+	"github.com/pokt-network/smt"
+
 	testsession "github.com/pokt-network/poktroll/testutil/session"
 	prooftypes "github.com/pokt-network/poktroll/x/proof/types"
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+const (
+	sumBytesSize       = 8
```

Comment:
We cannot, I removed the root size exports from the smt repo because their actual values are a function of the trie hasher which is used to produce any particular root (i.e., not constant).